<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="8" hostname="bLacKDoPh1n" failures="0" tests="6" name="Test default and locked out user for Log In and Log out" time="6.368" errors="0" timestamp="2024-07-01T15:59:42 IST">
  <testcase classname="base.BaseTest" name="@BeforeMethod beforeMethod" time="0.0">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.createTest(String)&amp;quot; because &amp;quot;this.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "this.extent" is null
at base.BaseTest.beforeMethod(BaseTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase classname="base.BaseTest" name="@AfterMethod afterMethod" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="base.BaseTest" name="@BeforeMethod beforeMethod" time="0.0">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.createTest(String)&amp;quot; because &amp;quot;this.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "this.extent" is null
at base.BaseTest.beforeMethod(BaseTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase classname="base.BaseTest" name="@AfterMethod afterMethod" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="base.BaseTest" name="@BeforeMethod beforeMethod" time="0.0">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.createTest(String)&amp;quot; because &amp;quot;this.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "this.extent" is null
at base.BaseTest.beforeMethod(BaseTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase classname="base.BaseTest" name="@AfterMethod afterMethod" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="base.BaseTest" name="@BeforeMethod beforeMethod" time="0.002">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.createTest(String)&amp;quot; because &amp;quot;this.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "this.extent" is null
at base.BaseTest.beforeMethod(BaseTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod beforeMethod -->
  <testcase classname="base.BaseTest" name="@AfterMethod afterMethod" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="com.project.test.TC_MultiWorkFlow" name="@AfterClass tearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase classname="com.project.test.TC_loginLogout" name="defaultLogInLogOut" time="0.833"/>
  <testcase classname="com.project.test.TC_loginLogout" name="logInWithLockedOutusr" time="0.472"/>
  <testcase classname="com.project.test.TC_PerformanceGlitchUser" name="testWithPerfUser" time="0.0">
    <skipped/>
  </testcase> <!-- testWithPerfUser -->
  <testcase classname="com.project.test.TC_ErrorUser" name="testWithErrlogin" time="0.0">
    <skipped/>
  </testcase> <!-- testWithErrlogin -->
  <testcase classname="com.project.test.TC_BrowserRefresh" name="testBrowserRefresh" time="0.0">
    <skipped/>
  </testcase> <!-- testBrowserRefresh -->
  <testcase classname="com.project.test.TC_MultiWorkFlow" name="multiFlow" time="0.0">
    <skipped/>
  </testcase> <!-- multiFlow -->
  <testcase classname="base.BaseTest" name="@BeforeTest mainBeforeTestMethod" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest mainBeforeTestMethod -->
  <testcase classname="base.BaseTest" name="@AfterSuite afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterTest -->
  <testcase classname="base.BaseTest" name="@BeforeTest mainBeforeTestMethod" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest mainBeforeTestMethod -->
  <testcase classname="base.BaseTest" name="@AfterSuite afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterTest -->
  <testcase classname="base.BaseTest" name="@AfterSuite afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterTest -->
  <testcase classname="base.BaseTest" name="@BeforeTest mainBeforeTestMethod" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest mainBeforeTestMethod -->
  <testcase classname="base.BaseTest" name="@BeforeTest mainBeforeTestMethod" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest mainBeforeTestMethod -->
  <testcase classname="base.BaseTest" name="@AfterSuite afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterTest -->
</testsuite> <!-- Test default and locked out user for Log In and Log out -->
